### PR TITLE
Resolve to ip to fix gRPC dns issues with cert

### PR DESCRIPTION
**Pull request by PWKad**
I tried the `tlsextraip` setting but was not seeing a difference so PRing this back to get feedback.

### CODE REVIEW

**Comment by alexbosworth** [comment]
Wouldn't this require an additional dependency?

**Comment by PWKad** [comment]
Yeah if the code sample looks OK I will clean up the pr and add that to package.json

Sent from Yahoo Mail on Android 
 
  On Sun, Oct 14, 2018 at 1:06 PM, Alex Bosworth<notifications@github.com> wrote:   
Wouldn't this require an additional dependency?

**Comment by PWKad** [comment]
Updated the PR

File: service/local_lnd.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 const dataDirName = 'data';
 const lndGrpcPort = process.env.LND_GRPC_PORT || 10009;
-const lndHost = process.env.LND_HOST || 'localhost';
 const {LNSERVICE_CHAIN} = process.env;
 const {LNSERVICE_LND_DIR} = process.env;
 const {LNSERVICE_NETWORK} = process.env;
 const tlsCertFileName = 'tls.cert';
 
+let lndHost = process.env.LND_HOST || 'localhost';
+if (!isIPAddress(lndHost)) {
```
Comment (alexbosworth) [inline service/local_lnd.js]: where is this function coming from?

Comment (PWKad) [reply service/local_lnd.js]: Sorry was throwing this together to get a PR up for review.  Updated with a regex to test for ip addresses

**Pull request merged**